psuutil/main.py

```diff
@@ -182,11 +183,14 @@ def status(index, textonly):
             msg = 'NOT PRESENT'
         status_table.append([psu_name, msg])
 
+    if not status_table:
+        return
+
     if textonly:
         for status in status_table:
             print status[0] + ':' + status[1]
     else:
-        print tabulate(status_table, header, tablefmt="grid")
+        print tabulate(status_table, header, tablefmt="simple")
```

Comment:
Use `click.echo` instead of `print`

Also please do the same on lines 135 and 153.

Reply:
Replaced all "print" to "click.echo"